### PR TITLE
Added inputfile parameter

### PR DESCRIPTION
**Pull request by jregalad**
Input file is read as positional parameter.

### CODE REVIEW

File: src/mypca.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 def main():
     sys.stderr.write("mypca V 1.0.0\n")
+    infilename = sys.argv[1]
```
Comment (7PintsOfCherryGarcia) [inline src/mypca.py:5]: Can we add an output file parameter as well?

Comment (jregalad) [reply src/mypca.py:5]: Sure, no problem!!

**Review by jregalad — COMMENTED** [review]
Output file as positional parameter added.

**Review by 7PintsOfCherryGarcia — COMMENTED** [review]
Looks great

**Comment by 7PintsOfCherryGarcia** [comment]
Merging changes by jregalad

**Pull request merged**